samples/projects/java-project/gradlew.bat

```diff
@@ -0,0 +1 @@
+../../../gradlew.bat
```

Comment:
If we stick with symlinks for the linux gradlew (still not a fan), we should at least turn this file into a proper batch file (instead of a symlink) that forwards everything to the root-level batch file. That way it will work properly for Windows users.

Reply:
I changed both to be independent files.